src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -662,10 +716,47 @@ public boolean isInGoal(int x, int y, int z) {
         }
         if (msg.equals("damn")) {
             logDirect("daniel");
+            return true;
+        }
+        if (msg.startsWith("add")){
+            String newCommands = msg.substring(3).trim();
+            commandsToBeExecuted = commandsToBeExecutedOld+ newCommands;
+            return true;
+        }
+        if (msg.equals("next")){
+            commandsToBeExecuted = commandsToBeExecutedOld;
```

Comment:
what is even going on with commandstobeexecutedold

Reply:
when you enter any commands ti will clear the upcoming commands unless the command entered is next or add. commandsToBeExecutedOld is used to restore the commandsToBeExecuted